avocado/core/test.py

```diff
@@ -280,6 +280,15 @@ def filename(self):
         else:
             return None
 
+    @property
+    def teststmpdir(self):
+        env_var = 'XXX_TESTS_COMMON_TMPDIR'
+        path = os.environ.get(env_var)
+        if path is None:
+            msg = 'Environment Variable %s is not set.' % env_var
+            raise EnvironmentError(msg)
```

Comment:
yep, I do think this should be included. Please don't forget to add documentation and docstring for it.